.github/workflows/mobile-docs.yml

```diff
@@ -29,14 +29,12 @@ jobs:
       packages: read
     runs-on: ${{ needs.env.outputs.agent_ubuntu }}
     timeout-minutes: 20
-    container:
-      image: ${{ needs.env.outputs.build_image_ubuntu }}
     steps:
     - uses: actions/checkout@v4
     - name: Add safe directory
       run: git config --global --add safe.directory "$GITHUB_WORKSPACE"
     - name: Generate docs
-      run: mobile/docs/build.sh
+      run: ./ci/run_envoy_docker.sh 'cd mobile && docs/build.sh'
```

Comment:
i haven't run mobile docs in a very long time, so more just curious if you tried this command locally and it worked?

Reply:
yeah - ive been running quite a bit for this PR - but previously also to test sphinx updates - one of the  motivations for this PR is to make that easier to maintain

Reply:
good shout here

i just reran it and realized i introduced an error at the end - checking with a local file server highlighed this - fixing now ...